bin/update

```diff
@@ -12,6 +12,30 @@ error()
   exit 1
 }
 
+# See if used attributes (first argument)
+# are valid according to given file (second argument)
+validate_content_field()
```

Comment:
I can't see this getting called.  Have you tried your changes?  You can clone the repo in your balab home directory and verify it by running `make`.  The results will be in the `balab` directory, which you can move to your `public_html` directory to view with a web browser.